examples/bootstrap/css-gridish/README.md

```diff
@@ -144,4 +144,4 @@ Thanks to flexbox’s wrapping functionality, nodes that specify rows are not ne
 
 ### What happens in the legacy implementation if I specify the column width for one breakpoint, but not the next larger breakpoint?
 
-To maintain a mobile-first opinion, column widths will scale to the next breakpoint if not specified. This means that a `.bootstrap-grid__col--sm--1` be the size of `.bootstrap-grid__col----2` if no `` class was declared.
```

Comment:
Hey @benxgao, these lines are automatically generated. Update the prefix in `css-gridish.json` of this directory. Then run `npm run build:examples` in the root directory of the project.